docs/bib/talks.bib

```diff
@@ -1,5 +1,16 @@
 % NB: entries with same author-title-year are not picked up:
 %     https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/117
+@unpublished{Heinrich20181205,
+  title  = {Gaussian Process Smoothing for shape systematics estimation},
```

Comment:
can we use the title form the slides? "Gaussian Process Shape  Estimation and Systematics"